src/external/lua/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
```

Comment:
I should use more verbose comments :)
What i meant is: that file is under ``src/external/`` -> probably all the changes would get overridden when updating bundled copy.
But as i now see, it looks like those 2 CMakeLists.txt are not from bundled libs, but was manually added like with rawspeed.
So all good, false alarm.